superset/viz.py

```diff
@@ -2040,7 +2040,7 @@ def get_properties(self, d):
             'radius': self.fixed_value if self.fixed_value else d.get(self.metric),
             'cat_color': d.get(self.dim) if self.dim else None,
             'position': d.get('spatial'),
-            '__timestamp': d.get('__timestamp'),
+            '__timestamp': d.get('__timestamp') or d.get('__time'),
```

Comment:
There's a constant in scope that you can use: `DTTM_ALIAS : d.get(DTTM_ALIAS) or d.get('__time'),`

Reply:
We could introduce a `DRUID_DTTM_ALIAS` constant but may be out of scope for this PR